main/src/library/Pkger/Pkger.flix

```diff
@@ -0,0 +1,92 @@
+use Core/Io/File.{canonical, exists, File, getParent, isDirectory, mkdirs, new, resolve};
+use Core/Io/IOError.IOError;
+use Result.flatMap;
+
+namespace Pkger/Pkger {
+    pub enum InitError {
+        case PathAlreadyExists(File)
+        case InitIOError(IOError)
+    }
+
+    //def IOErrorToInit(err: IOError): InitError {}
+
+    def mapIOErrorInit(r: Result[a, IOError]): Result[a, InitError] = Result.mapErr(err -> InitIOError(err), r)
+
+    ///
+    /// Create a new directory with called `name` in the current working directory and initialize a project called `name` inside of it.
+    ///
+    @test
+    pub def initPackage(name: String): Result[Unit, InitError] & Impure = {
```

Comment:
We need to figure out what types to use for the returned values because the `Result` type is not visible from Scala.

Reply:
I think the most appropriate might be something like returning a `java.util.Optional` of a Java string. If it is empty that means the operation was successful.